app.py

```diff
@@ -529,10 +694,7 @@ def onlinemasters(level, step):
         translations=TRANSLATIONS,
         version=version(),
         menu=None,
-        adventure_assignments=adventure_assignments,
-        loaded_program='',
-        loaded_program_name='',
-        adventure_name='')
+        loaded_program='')
```

Comment:
We should use the version in `test`, not this one.

Reply:
Fixed!